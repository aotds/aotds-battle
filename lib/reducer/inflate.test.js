"use strict";

var _inflate = require("./inflate");

var _inflate2 = _interopRequireDefault(_inflate);

function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }

const debug = require('debug')('aotds');

test('inflate', () => {
  let inflated = (0, _inflate2.default)({
    game: {},
    bogeys: [{
      id: 'enkidu',
      drive: 3,
      structure: {
        shields: [1, 1, 2]
      },
      weaponry: {
        firecons: 2
      }
    }]
  });
  expect(inflated).toMatchSnapshot();
});