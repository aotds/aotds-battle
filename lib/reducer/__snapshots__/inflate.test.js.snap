// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`inflate 1`] = `
Object {
  "bogeys": Object {
    "enkidu": Object {
      "drive": Object {
        "current": 3,
        "max": 3,
      },
      "id": "enkidu",
      "structure": Object {
        "shields": Object {
          "1": Object {
            "id": 1,
            "level": 1,
          },
          "2": Object {
            "id": 2,
            "level": 1,
          },
          "3": Object {
            "id": 3,
            "level": 2,
          },
        },
      },
      "weaponry": Object {
        "firecons": Object {
          "1": Object {
            "id": 1,
          },
          "2": Object {
            "id": 2,
          },
        },
      },
    },
  },
  "game": Object {},
}
`;
